src/main/resources/lode/swrl-module.xsl

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 Copyright (c) 2010-2013, Silvio Peroni <essepuntato@gmail.com>
+Copyright (C) 2023, Victor Chavez <vchavezb@protonmail.com>
```

Comment:
Can you please add your copyright under the code extension you have added? Otherwise it may look like you claim the copyright for the whole document, which I don't think it's appropriate

Reply:
Ok sure, yes it looks misleading.  Thanks for the feedback.